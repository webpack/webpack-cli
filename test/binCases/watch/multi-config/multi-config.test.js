"use strict";

jest.setTimeout(10E6);
/* eslint-disable node/no-unsupported-features  */
/* eslint-disable node/no-unsupported-features/es-syntax  */

const { runWatch } = require("../../../testUtils");

test("multi-config", async(done) => {
	const result = await runWatch(__dirname);
	const { stdout, stderr } = result;
	expect(stdout).toContain("");
	expect(stdout).toContain("webpack is watching the filesâ€¦");

	expect(stderr).toHaveLength(0);
	expect(stdout).toMatchSnapshot();
	done();
});
