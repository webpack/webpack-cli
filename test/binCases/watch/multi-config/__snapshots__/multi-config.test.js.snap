// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`multi-config 1`] = `
"Child
      Asset       Size  Chunks             Chunk Names
    main.js  945 bytes       0  [emitted]  main
    Entrypoint main = main.js
    [0] ./index.js 24 bytes {0} [built]
    
    WARNING in configuration
    The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
    You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/concepts/mode/
Child
      Asset       Size  Chunks             Chunk Names
    main.js  945 bytes       0  [emitted]  main
    Entrypoint main = main.js
    [0] ./index2.js 24 bytes {0} [built]
    
    WARNING in configuration
    The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
    You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/concepts/mode/
"
`;
